src/managedwriter/stream_connection.ts

```diff
@@ -107,20 +108,24 @@ export class StreamConnection extends EventEmitter {
   private handleError = (err: gax.GoogleError) => {
     this.trace('on error', err, JSON.stringify(err));
     if (this.shouldReconnect(err)) {
+      const retrySettings = this._writeClient['_retrySettings'];
       this.reconnect();
+      if (retrySettings.enableWriteRetries) {
+        this.resendAllPendingWrites();
+      } else {
+        err.message = 'reconnect triggered due to: ' + err.message;
```

Comment:
nit:  maybe something more developer friendly like "connection failed to service before response was received"?   Reconnect is mostly relevant to subsequent writes.

Reply:
yeah, I was already feeling like the wording was not nice here

Reply:
I would say message aborted due to failed connection, please retry the request.